tests/api/api.spec.ts

```diff
@@ -532,7 +532,74 @@ suite("@dataform/api", () => {
           tasks: [
             {
               type: "statement",
-              statement: "create or replace table `deeb.schema.name`  as select 1 as test"
+              statement: "create or replace table `deeb.schema.name` as select 1 as test"
```

Comment:
Whitespace change in bigquery adapter removes double space after `\`deeb.schema.name\``